code/Controllers/ContentController.php

```diff
@@ -58,6 +58,10 @@ class ContentController extends Controller
         'LoginForm'
     );
 
+    private static $casting = array(
```

Comment:
Super minor, but maybe we should use `[]` array syntax for SS4?

Reply:
I'm just using the same style as currently in the file - but I don't mind either way

Reply:
I've updated the code style for the file